### PR TITLE
[EI][Build]Updating EI with new binary and packages.

### PR DESCRIPTION
**Pull request by TusharKanekiDey**
*Issue #, if available:*

## Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

#### EIA Checklist
* When creating a PR:
- [ ] I've modified ```src/config/build_config.py``` in my PR branch by setting ```ENABLE_EI_MODE = True```
* When PR is reviewed and ready to be merged:
- [ ] I've reverted the code change on the config file mentioned above

* For reviewer, before merging, please cross-check:
- [ ] I've verified the code change on the config file mentioned above has already been reverted

*Description:*

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by YYStreet** [comment]
Is this one a replacement of https://github.com/aws/deep-learning-containers/pull/492?

**Comment by TusharKanekiDey** [comment]
> Is this one a replacement of #492?

Sorry, it should be a draft I was seeing if we could update some packages.

**Comment by TusharKanekiDey** [comment]
@ChuyangDeng Is it safe to remove sagemaker experiment from PT EIA Dockerfile like in #262 ?

**Comment by TusharKanekiDey** [comment]
@maaquib Can we update MMS Version in PT and MXNet EIA Dockerfiles?

File: pytorch/inference/docker/1.3.1/py3/Dockerfile.eia [hunk 1]
```diff
@@ -4,12 +4,12 @@ LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 LABEL dlc_major_version="1"
 
 # Add arguments to achieve the version, python and url
-ARG PYTHON_VERSION=3.6.6
+ARG PYTHON_VERSION=3.6.10
 ARG PYTORCH_VERSION=1.3.1
 ARG TORCHVISION_VERSION=0.4.2
 ARG GRAPHVIZ_VERSION=0.13.2
 ARG MMS_VERSION=1.0.8
```
Comment (maaquib) [inline pytorch/inference/docker/1.3.1/py3/Dockerfile.eia:11]: Can be upgraded to https://github.com/awslabs/multi-model-server/releases/tag/v1.1.2

Comment (TusharKanekiDey) [reply pytorch/inference/docker/1.3.1/py3/Dockerfile.eia:11]: Does EI Images support multi-model-server as well as vanilla Images ?

File: src/config/build_config.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Please only set it to True if you are preparing an EI related PR
 # Do remember to revert it back to False before merging any PR (including EI dedicated PR)
-ENABLE_EI_MODE = False
+ENABLE_EI_MODE = True
 # Frameworks for which you want to disable both builds and tests
-DISABLE_FRAMEWORK_TESTS = []
+DISABLE_FRAMEWORK_TESTS = ["mxnet","tensorflow"]
```
Comment (saimidu) [inline src/config/build_config.py:5]: If changes are being made to TensorFlow and MXNet dockerfiles, please enable their builds/tests here.

Comment (TusharKanekiDey) [reply src/config/build_config.py:5]: This is for temporary as we are trying to debug the PT failure issue currently.

File: mxnet/inference/docker/1.5.1/py3/Dockerfile.eia [hunk 3]
```diff
@@ -7,8 +7,8 @@ LABEL dlc_major_version="1"
 
 ARG MMS_VERSION=1.1.2
 ARG PYTHON=python3
-ARG PYTHON_VERSION=3.6.8
-ARG HEALTH_CHECK_VERSION=1.6.3
+ARG PYTHON_VERSION=3.6.10
+ARG HEALTH_CHECK_VERSION=s3://pytorch-eia-beta/openssl_fix/torch_eia-1.3.1-cp36-cp36m-manylinux1_x86_64.whl
```
[the remark this answers is not in the document]
Comment (TusharKanekiDey) [reply mxnet/inference/docker/1.5.1/py3/Dockerfile.eia:11]: Sorry, By mistake.

**Comment by TusharKanekiDey** [comment]
Outdated PR, All the Required PR are merged.

**Pull request closed without merging**